lib/players/human.rb

```diff
@@ -0,0 +1,8 @@
+class Human < Player
+
+  def move(board)
+    puts "Your turn #{self.token}! Choose a cell number (1-9):"
+    input = gets.strip
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Your turn #{self.token}! Choose a cell number (1-9):"
  gets.strip
end
```